src/lib/libast/tests/string/struniq.c

```diff
@@ -0,0 +1,34 @@
+#include "config_ast.h"  // IWYU pragma: keep
+
+#include <string.h>
+
+#include "ast.h"
+#include "terror.h"
+
+tmain() {
+    UNUSED(argc);
+    UNUSED(argv);
+
+    char *unsorted_strings[] = {"Line 9", "Line 8", "Line 7", "Line 6", "Line 5", "Line 4",
+                                "Line 3", "Line 2", "Line 1", "Line 9", "Line 8", "Line 7",
+                                "Line 6", "Line 5", "Line 4", "Line 3", "Line 2", "Line 1"};
+
+    char *unique_strings[] = {
+        "Line 1", "Line 2", "Line 3", "Line 4", "Line 5", "Line 6", "Line 7", "Line 8", "Line 9",
+    };
+
+    strsort(unsorted_strings, 18, strcmp);
```

Comment:
Replace the magic constant `18` with `sizeof(unsorted_strings) / sizeof(char *)`.